pom.xml

```diff
@@ -69,7 +69,7 @@
         <no-test-jar>false</no-test-jar>
         <useBeta>true</useBeta>
         <git-plugin.version>3.7.0</git-plugin.version>
-        <workflow-scm-step-plugin.version>2.6</workflow-scm-step-plugin.version>
+        <workflow-scm-step-plugin.version>2.7</workflow-scm-step-plugin.version>
```

Comment:
Strictly speaking, there is no need to update this to fix the issue, but we might as well bump to the lastest version while we're here so we can catch these kinds of issues before the PCT.